plugins/evernote.koplugin/main.lua

```diff
@@ -111,6 +125,127 @@ function EvernoteExporter:addToMainMenu(menu_items)
+                        keep_menu_open = true,
+                        callback = function()
+                            local MultiInputDialog = require("ui/widget/multiinputdialog")
+                            local auth_dialog
+                            auth_dialog = MultiInputDialog:new{
+                                title = _("Set authorization token for Joplin"),
+                                fields = {
+                                    {
+                                        text = self.joplin_token,
+                                        input_type = "string"
+                                    }
+                                },
+                                buttons = {
+                                    {
+                                        {
+                                            text = _("Cancel"),
+                                            callback = function()
+                                                UIManager:close(auth_dialog)
+                                            end
+                                        },
+                                        {
+                                            text = _("OK"),
+                                            callback = function()
+                                                local auth_field = auth_dialog:getFields()
+                                                --TODO do a check on auth string
+                                                self.joplin_token = auth_field[1]
+                                                self:saveSettings()
+                                                UIManager:close(auth_dialog)
+                                            end
+                                        }
+                                    }
+                                }
+                            }
+                            UIManager:show(auth_dialog)
+                            auth_dialog:onShowKeyboard()
+                        end
+                    },
+                    {
+                        text = _("Export to Joplin"),
+                        checked_func = function() return self.joplin_export end,
+                        callback = function()
+                            self.joplin_export = not self.joplin_export
+                            if self.joplin_export then
+                                self.html_export = false
+                                self.txt_export = false
+                            end
+                            self:saveSettings()
+                        end
+                    },
+                    {
+                        text = _("Help"),
+                        keep_menu_open = true,
+                        callback = function()
+                            UIManager:show(InfoMessage:new{
+                                text = T(_([[You can manually enter your auth token by editing %1/settings.reader.lua.
```

Comment:
Is that also where the Evernote settings are stored? Plugin-specific settings might be more appropriate.

Reply:
Yes, evernote login, notebook, default export type etc is stored there.

Reply:
These instructions are a bit vague. They should say which specific key needs to be added.